src/Account/components/SignatureRequestList.tsx

```diff
@@ -17,7 +17,6 @@ interface SignatureRequestListItemProps {
   onOpenTransaction?: (tx: Transaction, signatureRequest: MultisigTransactionResponse) => void
   signatureRequest: MultisigTransactionResponse
   style?: React.CSSProperties
-  testnet: boolean
```

Comment:
@troggy, why did you remove testnet parameter?

Reply:
because it is not needed anymore. Was needed for `TransactionList` props, where it was needed for `TransactionBuilder.fromXDR` call only. Now that we don't call `fromXDR` anymore, the prop is not needed neither in `TransactionList` nor in `SignatureRequestList`.